document-store/src/integrationTest/java/org/hypertrace/core/documentstore/DocStoreTest.java

```diff
@@ -334,12 +338,192 @@ public void testWithDifferentFieldTypes(String dataStoreName) throws Exception {
     Query queryStringField = new Query();
     filter = new Filter(Op.GT, "name", "abc1");
     queryStringField.setFilter(filter);
-    results = collection.search(queryBooleanField);
+    results = collection.search(queryStringField);
     documents = new ArrayList<>();
     while (results.hasNext()) {
       documents.add(results.next());
     }
-    Assertions.assertEquals(2, documents.size());
+    Assertions.assertEquals(3, documents.size());
+
+    datastore.deleteCollection(COLLECTION_NAME);
+  }
+
+  @ParameterizedTest
+  @MethodSource("databaseContextProvider")
+  public void testExistsOperatorWithSearch(String dataStoreName) throws Exception {
```

Comment:
How is this different from `testExistsFilter` test case?
https://github.com/hypertrace/document-store/blob/main/document-store/src/integrationTest/java/org/hypertrace/core/documentstore/DocStoreTest.java#L1960

Reply:
It is same. Will remove it.

Reply:
removed